source/includes/fundamentals/code-snippets/bson-extra-elements/BsonExtraElementsExample.java

```diff
@@ -0,0 +1,70 @@
+package fundamentals;
+
+import static com.mongodb.MongoClientSettings.getDefaultCodecRegistry;
+import static org.bson.codecs.configuration.CodecRegistries.fromProviders;
+import static org.bson.codecs.configuration.CodecRegistries.fromRegistries;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.bson.Document;
+import org.bson.codecs.configuration.CodecRegistry;
+import org.bson.codecs.pojo.PojoCodecProvider;
+
+import com.mongodb.client.MongoClient;
+import com.mongodb.client.MongoClients;
+import com.mongodb.client.MongoCollection;
+import com.mongodb.client.MongoDatabase;
+import com.mongodb.client.model.Filters;
+import com.mongodb.client.model.Updates;
+import com.mongodb.client.result.UpdateResult;
+
+public class BsonExtraElementsExample {
+
+	private static final String DOC_NAME_VALUE = "MDB0123";
```

Comment:
Have we standardized on 8 space indents?  4 is more typical.

Reply:
I think it's just a settings issue with my IDE; haven't had time to set the formatting. Will convert tabs to spaces and use 4 space indents.